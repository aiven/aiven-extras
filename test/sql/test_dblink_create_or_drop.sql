CREATE EXTENSION aiven_extras;
SELECT aiven_extras.dblink_slot_create_or_drop(format('dbname=%I port=%s', current_database(), current_setting('port')), 'test_slot', 'create');
SELECT slot_name from pg_replication_slots where slot_name = 'test_slot';
SELECT aiven_extras.dblink_slot_create_or_drop(format('dbname=%I port=%s', current_database(), current_setting('port')), 'test_slot', 'drop');
SELECT slot_name from pg_replication_slots where slot_name = 'test_slot';
