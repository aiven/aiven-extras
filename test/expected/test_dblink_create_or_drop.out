CREATE EXTENSION aiven_extras;
SELECT aiven_extras.dblink_slot_create_or_drop(format('dbname=%I port=%s', current_database(), current_setting('port')), 'test_slot', 'create');
 dblink_slot_create_or_drop 
----------------------------
 
(1 row)

SELECT slot_name from pg_replication_slots where slot_name = 'test_slot';
 slot_name 
-----------
 test_slot
(1 row)

SELECT aiven_extras.dblink_slot_create_or_drop(format('dbname=%I port=%s', current_database(), current_setting('port')), 'test_slot', 'drop');
 dblink_slot_create_or_drop 
----------------------------
 
(1 row)

SELECT slot_name from pg_replication_slots where slot_name = 'test_slot';
 slot_name 
-----------
(0 rows)

